mipengine/node/monetdb_interface/views.py

```diff
@@ -21,7 +21,7 @@ def get_view_data(context_id: str) -> List[List[Union[str, int, float, bool]]]:
 
 
 @validate_identifier_names
-def create_view(view_name: str, columns: List[str], datasets: List[str]):
+def create_view(view_name: str, pathology: str, datasets: List[str], columns: List[str], filters_json: str):
     cursor.execute(
-        f"CREATE VIEW {view_name} AS SELECT {', '.join(columns)} FROM data WHERE dataset IN ({str(datasets)[1:-1]})")
+        f"CREATE VIEW {view_name} AS SELECT {', '.join(columns)} FROM {pathology}_data WHERE dataset IN ({str(datasets)[1:-1]})")
```

Comment:
Good way to join a list: `','.join(lst)`
Bad way to join a list:   `str(lst)[1:-1]`

Reply:
It doesn't do the same thing...

This is doing the same thing `','.join('\'{0}\''.format(elem) for elem in list)` but is ugly as well. A bit better though.